pywinauto/controls/HwndWrapper.py

```diff
@@ -461,6 +462,33 @@ def send_message(self, message, wparam = 0, lparam = 0):
     # Non PEP-8 alias
     SendMessage = send_message
 
+    # -----------------------------------------------------------
+    def send_chars(self,
+                   message,
+                   with_spaces=True,
+                   with_tabs=True,
+                   with_newlines=True):
+        """
+        Silently send a string to the control
+
+        Parses modifiers Shift(+), Control(^), Menu(%) and Sequences like "{TAB}", "{Enter}"
+        For more information about Sequences and Modifiers navigate to SendKeysCtypes.py
+        """
+
+        keys = SendKeysCtypes.parse_keys(message, with_spaces, with_tabs, with_newlines)
+
+        for key in keys:
+            #keyinfo = key._get_key_info()
+            #res.append([key, keyinfo])
+
+            if (isinstance(key, SendKeysCtypes.EscapedKeyAction) or
+                    isinstance(key, SendKeysCtypes.VirtualKeyAction)):
+                key.Run()
+            else:
+                win32api.SendMessage(self.handle, win32con.WM_CHAR, key.get_key_scan_code(), 0)
```

Comment:
`lparam` is not used here, but it's important to send an info about modifiers etc. See [MSDN docs](https://msdn.microsoft.com/en-us/library/windows/desktop/ms646276(v=vs.85).aspx).